Data/ODBC/testsuite/src/ODBCSQLServerTest.cpp

```diff
@@ -417,6 +417,30 @@ catch(Poco::Exception& ex)
 }
 }
 
+void ODBCSQLServerTest::testUUIDsBulk()
```

Comment:
ok, but this test case will never run, unless it is added [here](https://github.com/pocoproject/poco/pull/4979/files#diff-b3e0f02c34e11ef60b0cdc953127453088585369e052aab5025361a513e96a1dL1120)